arch/x86/soc/intel_quark/quark_se/Kconfig.defconfig.series

```diff
@@ -17,13 +17,15 @@ config TOOLCHAIN_VARIANT
 	default "" if !X86_IAMCU
 
 config PHYS_RAM_ADDR
-	default 0xA8006400
+	default 0xA8007000 if X86_MMU
```

Comment:
The defaults chosen here were arbitrary. Let's just use A8007000 unconditionally.
Note that you have to also update the quark_se_c1000_ss under arch/arc/soc as well.